Signal/src/view controllers/CallViewController.swift

```diff
@@ -662,8 +662,6 @@ class CallViewController: UIViewController, CallObserver, CallServiceObserver, R
     func didPressVideo(sender: UIButton) {
         Logger.info("\(TAG) called \(#function)")
         let hasLocalVideo = !sender.isSelected
-        audioModeVideoButton.isSelected = hasLocalVideo
```

Comment:
The view state should be updated by observing changes to the model state.  It is, but we also have the convention of optimistically updating the view in these kinds of handlers.  That's hopefully redundant. Here, it was worse because the CallService will "reject" requests from the video to start video if the camera permissions are missing.